src/lib/Components/Inbox/Conversations/Message.tsx

```diff
@@ -53,15 +56,26 @@ const ImagePreviewContainer = styled.View`
   marginBottom: 10
 `
 
+const PDFPreviewContainer = styled.View`
+  marginBottom: 10
+`
+
 interface Props extends RelayProps {
   senderName: string
   artworkPreview?: JSX.Element
-  imagePreview?: JSX.Element
 }
 
 export class Message extends React.Component<Props, any> {
   render() {
     const date = moment(this.props.message.created_at).fromNow(true)
+    const attachments = this.props.message.attachments
+    let hasImageAttachment
+    let hasPDFAttachment
+    if (attachments.length > 0) {
+      hasImageAttachment = attachments[0].content_type === "image/jpeg"
```

Comment:
You could retrieve the actual attachment instead of a boolean to indicate it exists and get rid of some of that array access here and below, but dislike for array access is a personal dislike, so ¯\_(ツ)_/¯